scripts/release-build.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+const fs = require('fs').promises;
+const path = require('path');
+
+const directory = './build/';
+
+const verbatimFiles = ['LICENSE', 'README.md', 'index.d.ts', 'types.d.ts'];
+
+// Returns a string representing data ready for writing to JSON file
+function createDataBundle() {
+  const bcd = require('../index.js');
+  const string = JSON.stringify(bcd);
+  return string;
+}
+
+// Returns a promise for writing the data to JSON file
+async function writeData() {
+  const dest = path.resolve('./', directory, './data.json');
```

Comment:
```suggestion
  const dest = path.resolve(directory, 'data.json');
```

I didn't know that `path.resolve` could take any number of arguments and be like `path.join`, but since it is, this simplification seems possible.

Reply:
Done!